topic-configuration/README.adoc

```diff
@@ -0,0 +1,689 @@
+
+.Prerequisites
+* You are logged in to the {product} web console.
+* You have created a Kafka instance with at least one Kafka topic in {product}.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the topics that you want to configure.
+. In the *Topics* page, click the name of the topic that you want to review or edit.
++
+Alternatively, select the options icon (three vertical dots) for the relevant topic and click *Edit*.
+. Click *Edit properties*, review the current topic properties, and adjust any editable topic properties as needed.
+. Click *Save* to finish.
+
+[role="_additional-resources"]
+.Additional resources
+* {base-url}{getting-started-url}[_Getting started with {product}_^]
+* {base-url}{rhoas-cli-url}[_Getting started with the `rhoas` CLI_^]
+* {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^]
+
+
+[id="ref-supported-topic-properties_{context}"]
+== Supported topic properties in {product}
+
+[role="_abstract"]
+The following topic properties are supported in {product}. Each listed topic property indicates whether the property is editable or read only, and includes other relevant property attributes for your reference.
+
+=== Core configuration
+
+These properties determine the identity and core behavior of the topic. Before deploying your topic, consider entering all core configuration details for optimal results.
+
+Name::
++
+--
+The topic name is the unique identifier for the topic within the cluster. You need this to set up your producers and consumers, so make it something memorable.
+
+.Property attributes
+[cols="25%,75%"]
+|===
+h|Editable
+|Yes
+
+h|Type
+|String
+
+h|Default value
+|None
+
+h|Supported values
+|Letters (Aa-Zz), numbers (0-9), underscores ( _ ), or hyphens ( - ), maximum of 249 characters
+|===
+--
+
+Partitions::
++
+--
+Partitions are distinct lists of messages within a topic. Partitions are the main concurrency mechanism in Kafka and enable parts of a topic to be distributed over multiple brokers in the cluster. A topic can contain one or more partitions, enabling producer and consumer loads to be scaled. After you create a topic, you can increase the number of partitions but you cannot decrease it.
+
+.Property attributes
+[cols="25%,75%"]
+|===
+h|Editable
+|Yes (increase only)
+
+h|Type
+|Integer
+
+h|Default value
+|`1`
+
+h|Supported values
+|[`1,...`]
+
+h|Kafka property name
+|`num.partitions`
+|===
+--
+
+Replicas::
++
+--
+Replicas are copies of partitions in a topic. Partition replicas are distributed over multiple brokers in the cluster to ensure topic availability if a broker fails. When a follower replica is in sync with a partition leader, the follower replica can become the new partition leader if needed.
```

Comment:
Replication is **really** important for fault toleration and high availability. Do we want to underline this?

Reply:
Sure thing. How about this added line:

>Topic replication is an essential property for fault toleration and high availability.